src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -10,7 +10,7 @@ public class CliSyntax {
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
-    public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_EVENT_TAG = new Prefix("evt/");
```

Comment:
Do you think it would be better to change to this? I couldn't really tell if the code is referring to the event index or event itself at some parts of the code.
```suggestion
    public static final Prefix PREFIX_EVENT_INDEX_TAG = new Prefix("evt/");
```